portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,15 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Leah Ryu</h1>
+      <p>Hi! Welcome to my portfolio. My name is Leah and I'm a rising junior at Dartmouth College majoring in Computer Science and Studio Art. I'm interested in a lot of things--illustration/painting, string instruments, fencing, video games, books, CTF, and more. 
```

Comment:
Generally we want to avoid having super long lines. Not sure which style guide they asked you to follow; but https://developers.google.com/style/html-formatting asks for <80 chars per line. Same applies to .js file.
Shorter lines makes it easier to read (and avoid having to scroll horizontally in github).

Reply:
Got it, thanks